app/bundles/PageBundle/EventListener/PageImportExportSubscriber.php

```diff
@@ -0,0 +1,215 @@
+            'no_index'             => $page->getNoIndex(),
+            'lang'                 => $page->getLanguage(),
+            'variant_settings'     => $page->getVariantSettings(),
+            'uuid'                 => $page->getUuid(),
+        ];
+
+        $event->addEntity(Page::ENTITY_NAME, $pageData);
+        $this->logAction('export', $page->getId(), $pageData);
+    }
+
+    public function onPageImport(EntityImportEvent $event): void
+    {
+        if (Page::ENTITY_NAME !== $event->getEntityName() || !$event->getEntityData()) {
+            return;
+        }
+
+        $userName = '';
+        if ($event->getUserId()) {
+            $user     = $this->userModel->getEntity($event->getUserId());
+            $userName = $user ? $user->getFirstName().' '.$user->getLastName() : '';
+        }
+
+        $stats = [
+            EntityImportEvent::NEW    => ['names' => [], 'ids' => [], 'count' => 0],
+            EntityImportEvent::UPDATE => ['names' => [], 'ids' => [], 'count' => 0],
+        ];
+
+        foreach ($event->getEntityData() as $element) {
+            $page  = $this->entityManager->getRepository(Page::class)->findOneBy(['uuid' => $element['uuid']]);
+            $isNew = !$page;
+
+            $page ??= new Page();
+            $isNew && $page->setDateAdded(new \DateTime());
+            $page->setDateModified(new \DateTime());
+            $page->setUuid($element['uuid']);
+            $page->setTitle($element['title']);
+            $page->setIsPublished((bool) $element['is_published']);
+            $page->setAlias($element['alias'] ?? '');
+            $page->setTemplate($element['template'] ?? '');
+            $page->setContent($element['content'] ?? '');
+            $page->setCustomHtml($element['custom_html'] ?? '');
+            $page->setPublishUp($element['publish_up']);
+            $page->setPublishDown($element['publish_down']);
+            $page->setHits($element['hits']);
+            $page->setUniqueHits($element['unique_hits']);
+            $page->setVariantHits($element['variant_hits']);
+            $page->setRevision($element['revision']);
+            $page->setLanguage($element['lang'] ?? '');
+            $page->setMetaDescription($element['meta_description'] ?? '');
+            $page->setHeadScript($element['head_script'] ?? '');
+            $page->setFooterScript($element['footer_script'] ?? '');
+            $page->setRedirectType($element['redirect_type'] ?? '');
+            $page->setRedirectUrl($element['redirect_url']);
+            $page->setIsPreferenceCenter($element['is_preference_center'] ?? false);
+            $page->setNoIndex($element['no_index'] ?? false);
+            $page->setVariantSettings($element['variant_settings'] ?? []);
+
+            $isNew ? $page->setCreatedByUser($userName) : $page->setModifiedByUser($userName);
+
+            $this->entityManager->persist($page);
+            $this->entityManager->flush();
```

Comment:
We should be using models to store entities. It will also do some work for you like setting users and dates. But it also dispatches events that subscribers would miss this way.